azext_iot/smoke_tests/commands_test.ps1

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# Setting this environment variable enables the az command failures to be thrown and not swallowed
+
+$ErrorActionPreference = "Stop"
+
+$resource_group_name = "cli-int-test-rg"
```

Comment:
Can this be an env var so we can configure from the pipeline?

Reply:
Yes, was planning on making this update.